src/scheduled-delivery.js

```diff
@@ -123,6 +123,10 @@ export function selectDeliveryWindow(logisticsInfo, action) {
   }
 
   return logisticsInfo.map(li => {
+    if (!li) {
+      return li
```

Comment:
Does it make sense?

Reply:
@kevinch Normalmente não faria sentido o logisticsInfo ter um valor nulo no meio dele, mas foi a forma mais fácil de fazer a função `getNewLogisticsInfoWithScheduledDeliveryChoice`, porque eu queria permitir você filtrar quais itens teriam o seu sla alterado e o logisticsInfo se baseia na ordem dos itens em diversos momentos, então foi melhor só preencher os buracos que não eram pra serem alterados como `null`

Reply:
Bom, se faz sentido ok, mas ainda vejo `se o elemento não existir, retornar o elemento`.

Reply:
@kevinch Ah sim é apenas para manter: `[{...}, null, {...}]` da mesma forma e não: `[{...}, undefined, {...}]` ou qualquer outra representação de item vazio que não o original